src/components/Post/CommentHolder.jsx

```diff
@@ -15,7 +15,7 @@ const CommentHolder = ({
 }) => {
   return (
     <>
-      <ClickAwayListener onClickAway={() => setShowEmojis(false)}>
+      <ClickAwayListener onClickAway={() => {null}}>
```

Comment:
this is breaking the clickawaylistener, please fix them together!
Also this is not the correct approach here as background emoji dialog box is also opening. And this emoji box is not visible as you can see without scrolling
![image](https://github.com/narayan954/dummygram/assets/77617189/b4aa055d-5b0d-4344-b7fb-dd8c0399620e)